airbyte-integrations/connectors/source-google-ads/source_google_ads/schemas/ad_group_criterions.json

```diff
@@ -186,7 +192,7 @@
       "type": ["null", "string"]
     },
     "ad_group_criterion.url_custom_parameters": {
-      "type": ["null", "array"],
+      "type": ["null", "string", "array"],
```

Comment:
we could not support two data type for one field, better validate type and update it to one if needed